Source/cmake/OptionsWPE.cmake

```diff
@@ -235,6 +235,8 @@ if (COMPILER_IS_GCC_OR_CLANG AND UNIX AND NOT APPLE)
         set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -fno-stack-protector")
         set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fno-stack-protector")
     endif()
+    set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -Werror -Wformat=1 -Wno-pragmas")
```

Comment:
why not move to previous line which sets C/C++ compiler flags?

Reply:
OK I'll move it there